modules/core/src/v2/coins/utxo/recover.ts

```diff
@@ -241,6 +241,11 @@ export async function recover(coin: AbstractUtxoCoin, bitgo: BitGo, params: Reco
         return;
       }
 
+      if (addressType === Codes.UnspentTypeTcomb('p2tr') && !coin.supportsP2tr()) {
+        // P2tr is not supported. Skip.
```

Comment:
Could we add a `TODO: BG-<ticket>` reference here so we know to come back to this?

Reply:
I think this is just to exclude the coins that don't support `p2tr` so I don't think there's a relevant TODO here. This comment is probably just not worded great, I'll make it a bit clearer and also for the other if blocks I modeled this one off of.

Reply:
Got it - so just to be clear we don't need to do anything else to support recovery (signing with backup key) for txs spending taproot inputs?

Reply:
Not that I'm aware of although it might be worth taking a closer look at. The test case in this PR signs the user + backup script which is what we'd do for recovery, so functionally speaking it is definitely supported. But this if block was necessary because otherwise we'd test it for coins that don't support p2tr.